modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -166,6 +104,50 @@ You can list the possible services names by typing:
 ocis list
 ----
 
+== Default Paths
+
+=== Configuration Directory
+
+The default locations for _config_ files are:
+
+* For container images (inside the container) +
+`/etc/ocis/`
++
+* For binary releases +
+`$HOME/.ocis/config/`
++
+NOTE: You can deviate from the default location and define a custom configuration file location on startup using the environment variable `OCIS_CONFIG_DIR`.
++
+NOTE: When using a system user for the runtime which has no login and therefore no home directory like when used xref:deployment/binary/binary-setup.adoc#setting-up-systemd-for-infinite-scale[Setting up systemd for Infinite Scale], you *must* specify a configuration file location.
+
+=== Base Data Directory
+
+The base directory is important, because other services use this directory as base path if service dependent variables are not explicitly defined. Define them independently can be required when using a xref:deployment/container/orchestration/orchestration.adoc[Container Orchestration] setup.
```

Comment:
```suggestion
The base directory is important, because other services use this directory as base path if service dependent variables are not explicitly defined. Defining them independently can be required when using a xref:deployment/container/orchestration/orchestration.adoc[Container Orchestration] setup.
```